python/tvm/relax/block_builder.py

```diff
@@ -346,8 +351,12 @@ def emit_te(self, func: Callable, *args: Any, **kwargs: Any) -> Var:
 
         kwargs : Any, optional
             The keyword arguments passed to the function.
-            Note that the key "primfunc_name_hint" is reserved for passing name hint
-            to the PrimFunc that gets generated.
+            Note that the below keys are reserved:
+            * "primfunc_name_hint" is reserved for passing a name hint
+              to the PrimFunc that gets generated.
+            * "primfunc_public" is reserved for indicating whether the
```

Comment:
For emitting PrimFuncs from Relax, I assume private is the default we want. Happy to do it the other way if that's desirable.